net/mwan3/files/lib/mwan3/common.sh

```diff
@@ -63,19 +63,29 @@ mwan3_get_src_ip()
 	if [ "$family" = "ipv4" ]; then
 		addr_cmd='network_get_ipaddr'
 		default_ip="0.0.0.0"
-		sed_str='s/ *inet \([^ \/]*\).*/\1/;T; pq'
+		sed_str='s/ *inet \([^ \/]*\).*/\1/;T;p'
 		IP="$IP4"
 	elif [ "$family" = "ipv6" ]; then
 		addr_cmd='network_get_ipaddr6'
 		default_ip="::"
-		sed_str='s/ *inet6 \([^ \/]*\).* scope.*/\1/;T; pq'
+		sed_str='s/ *inet6 \([^ \/]*\).* scope.*/\1/;T;p'
```

Comment:
Please also add this to the new commit.

In my opinion, this belongs in a separate commit. I may want to backport it, as it fixes a problem that also exists in other OpenWrt branches.

Since I don't have a dependency on gnu sed, it's possible that busybox sed is also being used. Will that work as well? Could you please check that?